webpack.dev.config.js

```diff
@@ -29,7 +22,7 @@ const ethereumUrl =
 module.exports = {
   context: path.join(__dirname, 'src'),
   entry: ['react-hot-loader/patch', 'bootstrap-loader', 'index.js'],
-  devtool: 'eval',
+  devtool: 'source-map',
```

Comment:
https://webpack.js.org/configuration/devtool/
looks like source-map is not good for rebuilding, maybe switch to other one?

Reply:
`eval-source-map`, for example